src/segment-loader.js

```diff
@@ -776,7 +898,7 @@ export default class SegmentLoader extends videojs.EventTarget {
         seekable.start(0) < currentTime) {
       removeToTime = seekable.start(0);
     } else {
-      removeToTime = currentTime - 60;
+      removeToTime = currentTime - 30;
```

Comment:
deciding to go back to 60 second trim. Comment in code above stating `150mb` should actually be `150MB` which is ~5 minutes of playback, so we should be safe keeping 60 sec of back buffer